Binary files WRFV3.orig/test/em_b_wave/ideal.exe and WRFV3/test/em_b_wave/ideal.exe differ
diff -ru WRFV3.orig/test/em_b_wave/namelist.input WRFV3/test/em_b_wave/namelist.input
--- WRFV3.orig/test/em_b_wave/namelist.input	2012-02-17 19:15:22.000000000 +0100
+++ WRFV3/test/em_b_wave/namelist.input	2012-06-21 15:38:40.196207075 +0200
@@ -1,5 +1,5 @@
  &time_control
- run_days                            = 5,
+ run_days                            = 1,
  run_hours                           = 0,
  run_minutes                         = 0,
  run_seconds                         = 0,
@@ -11,7 +11,7 @@
  start_second                        = 00,   00,   00,
  end_year                            = 0001, 0001, 0001,
  end_month                           = 01,   01,   01,
- end_day                             = 05,   05,   05,
+ end_day                             = 01,   01,   01,
  end_hour                            = 00,   00,   00,
  end_minute                          = 00,   00,   00,
  end_second                          = 00,   00,   00,
Only in WRFV3/test/em_fire: grib2map.tbl
Only in WRFV3/test/em_fire: gribmap.txt
diff -ru WRFV3.orig/test/em_fire/hill_simple/namelist.input WRFV3/test/em_fire/hill_simple/namelist.input
--- WRFV3.orig/test/em_fire/hill_simple/namelist.input	2011-03-04 21:21:24.000000000 +0100
+++ WRFV3/test/em_fire/hill_simple/namelist.input	2012-06-21 15:59:08.760047708 +0200
@@ -1,8 +1,8 @@
  &time_control
  run_days                            = 0,
  run_hours                           = 0,
- run_minutes                         = 60,
- run_seconds                         = 0,
+ run_minutes                         = 0,
+ run_seconds                         = 10,
  start_year                          = 0001, 0001, 0001,
  start_month                         = 01,   01,   01,
  start_day                           = 01,   01,   01,
@@ -13,8 +13,8 @@
  end_month                           = 01,   01,   01,
  end_day                             = 01,   01,   01,
  end_hour                            = 00,   00,   00,
- end_minute                          = 60,   00,   00,
- end_second                          = 00,   00,   00,
+ end_minute                          = 00,   00,   00,
+ end_second                          = 10,   00,   00,
  history_interval_s                  = 120, 120,  120,
  frames_per_outfile                  = 1,     1,    1,
  restart                             = .false.,
Only in WRFV3/test/em_fire: ideal.exe
Only in WRFV3/test/em_fire: README.namelist
diff -ru WRFV3.orig/test/em_fire/two_fires/namelist.input WRFV3/test/em_fire/two_fires/namelist.input
--- WRFV3.orig/test/em_fire/two_fires/namelist.input	2011-03-04 21:21:24.000000000 +0100
+++ WRFV3/test/em_fire/two_fires/namelist.input	2012-06-21 15:51:43.368611686 +0200
@@ -1,7 +1,7 @@
  &time_control
  run_days                            = 0,
  run_hours                           = 0,
- run_minutes                         = 60,
+ run_minutes                         = 1,
  run_seconds                         = 0,
  start_year                          = 2006, 0001, 0001,
  start_month                         = 01,   01,   01,
@@ -12,8 +12,8 @@
  end_year                            = 2006, 0001, 0001,
  end_month                           = 01,   01,   01,
  end_day                             = 01,   01,   01,
- end_hour                            = 10,   00,   00,
- end_minute                          = 00,    0,   0,
+ end_hour                            = 09,   00,   00,
+ end_minute                          = 01,    0,   0,
  end_second                          = 0,    00,   00,
  history_interval_s                  = 120,  30,   30,
  frames_per_outfile                  = 1,     1,    1,
Only in WRFV3/test/em_fire: wrf.exe
Binary files WRFV3.orig/test/em_heldsuarez/ideal.exe and WRFV3/test/em_heldsuarez/ideal.exe differ
diff -ru WRFV3.orig/test/em_heldsuarez/namelist.input WRFV3/test/em_heldsuarez/namelist.input
--- WRFV3.orig/test/em_heldsuarez/namelist.input	2012-02-17 19:15:22.000000000 +0100
+++ WRFV3/test/em_heldsuarez/namelist.input	2012-06-21 16:03:49.531101639 +0200
@@ -1,5 +1,5 @@
  &time_control
- run_days                            = 1000,
+ run_days                            = 1,
  run_hours                           = 0,
  run_minutes                         = 0,
  run_seconds                         = 0,
@@ -9,9 +9,9 @@
  start_hour                          = 00,   00,   00,
  start_minute                        = 00,   00,   00,
  start_second                        = 00,   00,   00,
- end_year                            = 0002, 0010, 0010,
+ end_year                            = 0001, 0010, 0010,
  end_month                           = 01,   01,   01,
- end_day                             = 01,   01,   01,
+ end_day                             = 02,   01,   01,
  end_hour                            = 00,   00,   00,
  end_minute                          = 00,   00,   00,
  end_second                          = 00,   00,   00,
Binary files WRFV3.orig/test/em_les/ideal.exe and WRFV3/test/em_les/ideal.exe differ
diff -ru WRFV3.orig/test/em_les/namelist.input WRFV3/test/em_les/namelist.input
--- WRFV3.orig/test/em_les/namelist.input	2012-02-17 19:15:22.000000000 +0100
+++ WRFV3/test/em_les/namelist.input	2012-06-21 16:11:16.992395929 +0200
@@ -1,7 +1,7 @@
  &time_control
  run_days                            = 0,
- run_hours                           = 1,
- run_minutes                         = 00,
+ run_hours                           = 0,
+ run_minutes                         = 2,
  run_seconds                         = 00,
  start_year                          = 0001, 0001, 0001,
  start_month                         = 01,   01,   01,
@@ -12,8 +12,8 @@
  end_year                            = 0001, 0001, 0001,
  end_month                           = 01,   01,   01,
  end_day                             = 01,   01,   01,
- end_hour                            = 01,   02,   00,
- end_minute                          = 00,   30,   00,
+ end_hour                            = 01,   01,   00,
+ end_minute                          = 00,   32,   00,
  end_second                          = 00,   00,   00,
  history_interval_m                  = 60,   10,   1,
  history_interval_s                  = 00,   00,   1,
Binary files WRFV3.orig/test/em_quarter_ss/ideal.exe and WRFV3/test/em_quarter_ss/ideal.exe differ
diff -ru WRFV3.orig/test/em_quarter_ss/namelist.input WRFV3/test/em_quarter_ss/namelist.input
--- WRFV3.orig/test/em_quarter_ss/namelist.input	2012-02-17 19:15:22.000000000 +0100
+++ WRFV3/test/em_quarter_ss/namelist.input	2012-06-21 16:18:15.319325415 +0200
@@ -1,7 +1,7 @@
  &time_control
  run_days                            = 0,
  run_hours                           = 0,
- run_minutes                         = 60,
+ run_minutes                         = 5,
  run_seconds                         = 0,
  start_year                          = 0001, 0001, 0001,
  start_month                         = 01,   01,   01,
@@ -13,7 +13,7 @@
  end_month                           = 01,   01,   01,
  end_day                             = 01,   01,   01,
  end_hour                            = 00,   00,   00,
- end_minute                          = 120,  120,  120,
+ end_minute                          = 5,    5,    5,
  end_second                          = 00,   00,   00,
  history_interval                    = 30,   10,   10,
  frames_per_outfile                  = 1000, 1000, 1000,
Binary files WRFV3.orig/test/em_scm_xy/ideal.exe and WRFV3/test/em_scm_xy/ideal.exe differ
diff -ru WRFV3.orig/test/em_scm_xy/namelist.input WRFV3/test/em_scm_xy/namelist.input
--- WRFV3.orig/test/em_scm_xy/namelist.input	2012-02-17 19:15:22.000000000 +0100
+++ WRFV3/test/em_scm_xy/namelist.input	2012-06-21 16:23:24.475698943 +0200
@@ -1,6 +1,6 @@
  &time_control
  run_days                            = 0,
- run_hours                           = 59,
+ run_hours                           = 3,
  run_minutes                         = 0,
  run_seconds                         = 0,
  start_year                          = 1999,
@@ -11,8 +11,8 @@
  start_second                        = 00,
  end_year                            = 1999,
  end_month                           = 10,
- end_day                             = 25,
- end_hour                            = 06,
+ end_day                             = 22,
+ end_hour                            = 22,
  end_minute                          = 00,
  end_second                          = 00,
  history_interval                    = 60,
Binary files WRFV3.orig/test/em_tropical_cyclone/ideal.exe and WRFV3/test/em_tropical_cyclone/ideal.exe differ
diff -ru WRFV3.orig/test/em_tropical_cyclone/namelist.input WRFV3/test/em_tropical_cyclone/namelist.input
--- WRFV3.orig/test/em_tropical_cyclone/namelist.input	2012-02-17 19:15:22.000000000 +0100
+++ WRFV3/test/em_tropical_cyclone/namelist.input	2012-06-21 15:28:40.587473802 +0200
@@ -1,6 +1,6 @@
  &time_control
- run_days                            = 6,
- run_hours                           = 0,
+ run_days                            = 0,
+ run_hours                           = 1,
  run_minutes                         = 0,
  run_seconds                         = 0,
  start_year                          = 2007,
@@ -11,8 +11,8 @@
  start_second                        = 00,
  end_year                            = 2007,
  end_month                           = 09,
- end_day                             = 7,
- end_hour                            = 0,
+ end_day                             = 1,
+ end_hour                            = 1,
  end_minute                          = 00,
  end_second                          = 00,
  history_interval                    = 1440,
Only in WRFV3/test/nmm_tropical_cyclone: ideal.exe
Only in WRFV3/test/nmm_tropical_cyclone: README.namelist
Only in WRFV3/test/nmm_tropical_cyclone: wrf.exe
